src/css/theme.css

```diff
@@ -0,0 +1,226 @@
+@theme {
```

Comment:
How you feeling about having all of these values pulled from a Figma file one day?

Reply:
Any thoughts on having a core and semantic layer?

Reply:
@sherakama I could for sure, perhaps UComm @iamrentman could work with us to define a set of semantic variables?